.github/ISSUE_TEMPLATE.md

```diff
@@ -1,58 +1,20 @@
-#### Do NOT include any confidential information, this issue will be created on a public repo
-#### This section is completed by the issue reporter
-##### On which version(s) do you encounter this issue?
-###### Please provide the output to 'git describe --all' for  all deployments where you have confirmed it occurs.
-
-##### What type of issue is this? Please tick the appropriate box.
```

Comment:
I prefer the former. Features and bugs would be better tracked within rpc-openstack since it already connects to the waffle board.